base/libgit2/utils.jl

```diff
@@ -26,7 +26,7 @@ function version()
     minor = Ref{Cint}(0)
     patch = Ref{Cint}(0)
     ccall((:git_libgit2_version, :libgit2), Void,
-          (Ptr{Cint}, Ptr{Cint}, Ptr{Cint}), major, minor, patch)
+          (Ref{Cint}, Ref{Cint}, Ref{Cint}), major, minor, patch)
```

Comment:
Perhaps, but since this is related to `version` might as well just do it here. THis is in any case a very trivial change.